styles.css

```diff
@@ -0,0 +1,181 @@
+* {
+	box-sizing: border-box;
+}
+
+html {
+	font-size: 22px;
+	font-family: 'Orbitron', sans-serif;
+	font-weight: 600;
+}
+
+body {
+	background-color: #120458;
+	color: #FE75FE;
+	margin-top: 60px;
+	background-image: url(BG.png);
+	background-repeat: no-repeat;
+	background-size: cover;
+	image-rendering: pixelated;
+	background-position: center;
+	background-attachment: fixed;
+}
+
+nav {
+	position: fixed;
+	background-color: #120458;
+	top: 0;
+	width: 100%;
+	color: #FF00A0;
```

Comment:
Just a recommendation... depending on what accessibility level you are after, this colour does not pass AAA standards in normal text.  Your nav text is larger so this is actually fine.

https://webaim.org/resources/contrastchecker/

Reply:
Thank you for recommendation. I will not be using this colour in normal text. However, I have now changed the hover colour in my About Me section (to white) to meet this recommendation.

Reply:
> Thank you for recommendation. I will not be using this colour in normal text. However, I have now changed the hover colour in my About Me section (to white) to meet this recommendation.

Great!